smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -132,16 +134,25 @@ public async Task Configure(string connectionString, string image, string hostna
                         cts.Token);
                 }
 
-                string args = $". {this.scriptDir}\\IotEdgeSecurityDaemon.ps1; Install-SecurityDaemon -Manual " +
-                              $"-ContainerOs Windows -DeviceConnectionString '{connectionString}' -AgentImage '{image}'";
-
-                this.proxy.ForEach(proxy => { args += $" -Proxy '{proxy}'"; });
+                string args;
+                if (this.isCleanInstall)
+                {
+                    args = $". {this.scriptDir}\\IotEdgeSecurityDaemon.ps1; Install-SecurityDaemon -Manual " +
+                           $"-ContainerOs Windows -DeviceConnectionString '{connectionString}' -AgentImage '{image}'";
 
-                if (!string.IsNullOrEmpty(this.offlineInstallationPath))
+                    if (!string.IsNullOrEmpty(this.offlineInstallationPath))
+                    {
+                        args += $" -OfflineInstallationPath '{this.offlineInstallationPath}'";
+                    }
+                }
+                else
                 {
-                    args += $" -OfflineInstallationPath '{this.offlineInstallationPath}'";
+                    args = $". {this.scriptDir}\\IotEdgeSecurityDaemon.ps1; Initialize-IoTEdge -Manual " +
+                           $"-ContainerOs Windows -DeviceConnectionString '{connectionString}' -AgentImage '{image}'";
                 }
 
+                this.proxy.ForEach(proxy => { args += $" -Proxy '{proxy}'"; });
```

Comment:
I am not sure, does Initialize-IoTEdge also support -Proxy parameter?

Reply:
_Install-IoTEdge_ is a wrapper arround _Deploy-IoTEdge_ and _Initialize-IoTEdge_. Only the first two use _-Proxy_ parameter and _Initialize-IoTEdge_ does not.
I updated the code to not pass the _-Proxy_ parameter to _Initialize-IoTEdge_.